Subject: [PATCH] [ActivityManager]an extra animation when finish a visible paused activity

ActivityA:
The attr windowIsTranslucent is set true
ActivityB:
a normal activity, and when paused, then post a 3 seconds message to
finish itself

start ActivityB first and then startActivityA with adb commands, After
ActivityA is resumed, and an extra animation of activity B is excuted
It is a bad UE

Test: two demo apps with activitya and activityb which is attached in
the issue 62047246

diff --git a/services/core/java/com/android/server/am/ActivityStack.java b/services/core/java/com/android/server/am/ActivityStack.java
index 998e2c6..2ccbae3 100644
--- a/services/core/java/com/android/server/am/ActivityStack.java
+++ b/services/core/java/com/android/server/am/ActivityStack.java
@@ -3593,9 +3593,7 @@
             // it is done pausing; else we can just directly finish it here.
             if (DEBUG_PAUSE) Slog.v(TAG_PAUSE, "Finish not pausing: " + r);
             if (r.visible) {
-                mWindowManager.prepareAppTransition(transit, false);
                 mWindowManager.setAppVisibility(r.appToken, false);
-                mWindowManager.executeAppTransition();
                 if (!mStackSupervisor.mWaitingVisibleActivities.contains(r)) {
                     mStackSupervisor.mWaitingVisibleActivities.add(r);
                 }
